### PR TITLE
Feat: Introduced IGM Implementation

### PR DESCRIPTION
**Pull request by sharmasahil0331**

### CODE REVIEW

File: deploying_ansible/ansible_hosts [hunk 1]
```diff
@@ -11,10 +14,11 @@
...
+[testing:vars]
 [stage:vars]
 [preprod:vars]
 [legacy:vars]
 [prod:vars]
-ansible_ssh_user=ubuntu
-ansible_ssh_private_key_file=/Users/navdeepagarwal/.ondc/ondc-prod-prod-ondc-ec2-key-pair-ssh-key
-ansible_python_interpreter=/usr/bin/python3
\ No newline at end of file
+ansible_ssh_user = ubuntu
```
Comment (navdeep710) [inline deploying_ansible/ansible_hosts:22]: we can remove this

File: docker-compose-for-local.yaml [hunk 2]
```diff
@@ -107,18 +106,79 @@ services:
...
+      - 8989
+    depends_on:
+      - mongo
+    volumes:
+      - "./images:/app/images"
+    restart: always
+
   py-ondc-protocol:
-    image: navdeep710/ondc-protocol-layer-server:pr-sse-v7.8
+    image: adityapatil123/ondc-python-protocol-layer:pr-sse-v7.4.2
```
Comment (navdeep710) [inline docker-compose-for-local.yaml:137]: use latest one
navdeep710/ondc-protocol-layer-server:pr-sse-v7.8

File: docker-compose-for-local.yaml [hunk 3]
```diff
@@ -107,18 +106,79 @@ services:
...
     container_name: py-ondc-protocol
     environment:
-      ENV: ${PROTOCOL_ENV}
+      ENV: prod
```
Comment (navdeep710) [inline docker-compose-for-local.yaml:140]: can we read this from variables

**Review by navdeep710 — COMMENTED** [review]
please check for comments

**Comment by sharmasahil0331** [comment]
Comments Resolved @navdeep710

**Review by navdeep710 — APPROVED** [review]

**Pull request merged**